GitUI/CommandsDialogs/FormCommit.cs

```diff
@@ -307,6 +307,11 @@ public FormCommit(GitUICommands commands, CommitKind commitKind = CommitKind.Nor
             selectionFilter.Size = DpiUtil.Scale(selectionFilter.Size);
             toolStripStatusBranchIcon.Width = DpiUtil.Scale(toolStripStatusBranchIcon.Width);
 
+            if (!GitVersion.Current.SupportStashStaged)
```

Comment:
Required for WSL Git. 
GitVersion.Current is to be used when Windows Git is always used or in settings etc when Module is not available.
```suggestion
            if (!Module.GitVersion.SupportStashStaged)
```

Reply:
Done.